app/common/gristUrls.ts

```diff
@@ -732,6 +732,10 @@ export interface GristLoadConfig {
   // on this for custom domains, but set it generally for all pages.
   org?: string;
 
+  // True if there are no subdomains  or separate servers defined for
+  // the home servers or doc workers.
```

Comment:
```suggestion
  // Makes the Grist frontend access the Grist instance using its current URL in the browser, rather than APP_HOME_URL.
  // Used to simplify setup of single-domain (no subdomain / doc worker) installations.
```

Tried to rephrase this to describe what the config option does, rather than when it's true / false. That's because the logic for it being true / false is defined elsewhere, could easily change later, and we'd forget to update this comment.